docs/site/migration/auth/example.md

```diff
@@ -297,93 +297,95 @@ To make the tutorial concise, the code details is omitted here. You can find the
 ACLs and how the endpoints are decorated in file
 [src/controller/project.controller.ts](https://github.com/strongloop/loopback-next/tree/master/examples/access-control-migration/src/controllers/project.controller.ts)
 
-Next, we write the authorizer that calls casbin enforcers to make the decision
+Next, we write the authorizer that calls casbin enforcers to make the decision.
 
 1. Create an authorizer that retrieves the authorization metadata from context,
    then execute casbin enforcers to make decision.
 
-_If you are not familiar with the concept authorizer, you can learn it in the
-document
-[Programming Access Policies](../../Loopback-component-authorization#programming-access-policies)_
+   _If you are not familiar with the concept authorizer, you can learn it in the
+   document
+   [Programming Access Policies](../../Loopback-component-authorization#programming-access-policies)_
 
-The complete authorizer file can be found in
-[components/casbin-authorization/services/casbin.authorizer.ts](https://github.com/strongloop/loopback-next/blob/master/examples/access-control-migration/src/components/casbin-authorization/services/casbin.authorizer.ts).
-It retrieves the three required fields from the authorization context: subject
-from `principal`, `resource` as object, and action, to invoke casbin enforcers
-and make decision.
+   The complete authorizer file can be found in
+   [components/casbin-authorization/services/casbin.authorizer.ts](https://github.com/strongloop/loopback-next/blob/master/examples/access-control-migration/src/components/casbin-authorization/services/casbin.authorizer.ts).
+   It retrieves the three required fields from the authorization context:
+   subject from `principal`, `resource` as object, and action, to invoke casbin
+   enforcers and make decision.
 
-2. Create a voter for instance level endpoints to append the project id to the
```

Comment:
Why do we change them all to`1.`?

Reply:
I think either incremental number or all as 1. works.
See example in https://loopback.io/doc/en/lb4/Authentication-Tutorial.html#try-it-out

The real fix for the listing is the indence, so the number doesn't quite matter. But since the number causes confusion I revert it.